tests/java/verifiers/src/main/java/io/vectorized/reads_writes/Workload.java

```diff
@@ -0,0 +1,437 @@
+package io.vectorized.reads_writes;
+import java.lang.Thread;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.Properties;
+import java.util.Queue;
+import java.util.concurrent.Semaphore;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.clients.producer.Callback;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.clients.producer.RecordMetadata;
+import org.apache.kafka.common.TopicPartition;
+
+public class Workload {
+  static enum OpStatus { WRITING, UNKNOWN, WRITTEN, SEEN, SKIPPED }
+
+  static class OpInfo {
+    public long id;
+    public long started_us;
+    public OpStatus status;
+    public int partition;
+    public long offset;
+  }
+
+  static class WriterStat {
+    public long countWritten = 0;
+    public long countRead = 0;
+    public long lastOffset = -1;
+    public int inflights = 0;
+    public boolean draining = false;
+    public Semaphore drained = new Semaphore(0);
+  }
+
+  static volatile long started_s = -1;
+  static int workload_id_gen = 0;
+  static synchronized int get_workload_id() { return workload_id_gen++; }
```

Comment:
can we stick to the same naming convention for all functions/methods i.e. camelCase ?